docs/Learning-Environment-Design-Agents.md

```diff
@@ -933,7 +933,8 @@ hyperparameter hierarchy in both.
 
 Cooperative behavior in ML-Agents can be enabled by instantiating a `SimpleMultiAgentGroup`,
 typically in an environment controller or similar script, and adding agents to it
-using the `RegisterAgent(Agent agent)` method. Using `MultiAgentGroup` enables the
+using the `RegisterAgent(Agent agent)` method. Note that all agents added to the same `MultiAgentGroup`
```

Comment:
I think we only have `SimpleMultiAgentGroup` and `IMultiAgentGroup`, no `MultiAgentGroup`. (To verify)

Reply:
Confirmed and fixed